src/output/icons.rs

```diff
@@ -222,6 +231,7 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     "Dockerfile"          => Icons::DOCKER,         // 
     "Earthfile"           => '\u{f0ac}',            // 
     "environment"         => Icons::CONFIG,         // 
+    "fstab"               => Icons::DATABASE,       // 
```

Comment:
I am not sure I like this one, the icon makes me think of a binary format, not the actual plain text fstab

```
# <device>                                <dir> <type> <options> <dump> <fsck>
UUID=0a3407de-014b-458b-b5c1-848e92a327a3 /     ext4   defaults  0      1
UUID=f9fe0b69-a280-415d-a03a-a32752370dee none  swap   defaults  0      0
UUID=b411dc99-f0a0-4c87-9e05-184977be8539 /home ext4   defaults  0      2
```

Reply:
It's all good, maybe we could go with generic config gear `\u{e615}`, or the hard disk icon `\u{f0a0}`? I was just thinking `fstab` -> filesystem -> disk